taichi/ir/ir.cpp

```diff
@@ -20,6 +20,10 @@ IRBuilder &current_ast_builder() {
   return context->builder();
 }
 
+CompileConfig &IRNode::get_config() const {
+  return get_kernel()->program.config;
+}
```

Comment:
Nice to have this! And BTW I like your proposal in #455 about a per-kernel compile config, I believe that's how we eventually entirely obsolete ti.cfg. This helper would be helpful when supporting that as well.

Reply:
Yes, I believe we may finally use `get_kernel()->config` for this.

Reply:
Agreed!